ui/react-components/helper.js

```diff
@@ -33,3 +33,23 @@ export const isServiceTypeEnabled = appConfig => {
 export const getValidProviders = providers => {
     return providers && providers.filter(provider => provider.response === PROVIDER_RESPONSES.ACCEPTED);
 };
+
+export const searchFeildOnChangeHandler=(state,setState,selectedState,setSelectedState,e)=>{
```

Comment:
Assuming here `searchFeildOnChangeHandler` is a type and should have been `searchFieldOnChangeHandler`. Similar for `searchFeildOnRemoveHandler`.

Reply:
can you please elaborate more. We are not getting exactly what you are saying

Reply:
Just suggesting a name change. I think the `Feild` should be `Field`. Let me know if that is not the case.

Reply:
Ok. You are right. Will update the name

Reply:
Thanks